tools/crm_resource.c

```diff
@@ -671,7 +675,8 @@ attr_set_type_cb(const gchar *option_name, const gchar *optarg, gpointer data, G
         options.attr_set_type = XML_TAG_META_SETS;
     } else if (pcmk__str_any_of(option_name, "-z", "--utilization", NULL)) {
         options.attr_set_type = XML_TAG_UTILIZATION;
-    }
+    } else if (pcmk__str_any_of(option_name, "-y", "--element", NULL)) {
+        options.attr_set_type = XML_TAG_ELEMENT;
```

Comment:
This won't be an actual XML tag, just a magic string that doesn't match any real tag, so you can just define a constant for it in this file